src/TestProblem/Hydro/Riemann/Init_TestProb_Hydro_Riemann.cpp

```diff
@@ -140,96 +152,99 @@ void SetParameter()
    {
       case SOD_SHOCK_TUBE : Riemann_RhoL = 1.0;    Riemann_VelL = 0.0;  Riemann_PreL = 1.0;  Riemann_VelL_T1 = 0.0;  Riemann_VelL_T2 = 0.0;
                             Riemann_RhoR = 0.125;  Riemann_VelR = 0.0;  Riemann_PreR = 0.1;  Riemann_VelR_T1 = 0.0;  Riemann_VelR_T2 = 0.0;
-                            Riemann_EndT = 0.1;
+                            Riemann_EndT = 0.1; ;  Riemann_Pos = 0.5;
```

Comment:
Is it intended to read `Riemann_Pos` as a parameter but then set it here?

Reply:
Yes, the user-defined condition allows us to specify the discontinuity location, while all the other built-in test reference solutions are set at 0.5.

Reply:
Got it!